setup.py

```diff
@@ -131,6 +131,12 @@ def is_requirement(line):
     entry_points={
         'lms.djangoapp': [
             "learning_assistant = learning_assistant.apps:LearningAssistantConfig"
+        ],
+        'cms.djangoapp': [
```

Comment:
Because the `CourseApps` API is registered under the CMS, the `Learning Assistant` needs to be registered as a plugin there as well. Otherwise, the plugin is not included in the CMS's `INSTALLED_APPS` list, which causes a runtime error, because the `LearningAssistantCourseApp` plugin refers to the `Learning Assistant` models, which are not available in the CMS.